backend/src/utils.ts

```diff
@@ -15,25 +15,6 @@ const limiter = new Bottleneck({
   maxConcurrent: 5
 });
 
-// https://github.com/patriksimek/vm2/issues/268#issuecomment-593536163
-const vm2SecurityPatch = (vm: VM): void => {
```

Comment:
it's now merged: https://github.com/patriksimek/vm2/issues/268#issuecomment-602115904